values.yaml

```
@@ -465,6 +488,24 @@ connectInject:
   #     namespace-label: label-value
   namespaceSelector: null
 
+  # allowNamespacesList is a list of k8s namespaces to allow Connect sidecar
+  # injection in. If a k8s namespace is not included or is listed in `denyNamespacesList`,
+  # pods in that k8s namespace will not be injected even if they are explicitly
+  # annotated. Use "*" to automatically allow all k8s namespaces.
+  # For example, ["namespace1", "namespace2"] will only allow pods in the k8s
+  # namespaces `namespace1` and `namespace2` to have Connect sidecars injected
+  # and registered with Consul. All other k8s namespaces will be ignored.
+  # Note: `denyNamespacesList` takes precedence over values defined here and
+  # `namespaceSelector` takes precedence over both since it is applied first.
+  # Requires consul-k8s v0.x+
+  allowNamespacesList: ["*"]
```

```suggestion
  k8sAllowNamespaces: ["*"]
```
k8s to make it clear we're talking k8s namespaces (in inject that doesn't matter but it will match sync)
removing List because that's inferred from the datatype

---

Updated.